mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -144,13 +144,11 @@
     <!-- Button in the search view that lets a user search by scanning a QR code -->
     <string name="search_scan_button">Scan</string>
     <!-- Button in the search view that lets a user search by using a shortcut -->
```

Comment:
Not blocking merge on this, but shouldn't the comment be updated here since the string doesn't seem to reflect the comment?
CC @boek